### PR TITLE
support node 10 by upgrading unzip

### PR DESCRIPTION
**Pull request by michaelglass**
should outfox #11 

also, mildly helps with https://github.com/elm-lang/elm-platform/issues/231

issue described therein.

Because I couldn't reproduce the error locally to this test suite, I wanted a different way to verify that this works. So I added the new binwrap here: https://github.com/NoRedInk/noredink-ui/pull/48/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R26

and then my error therein was resolved.

### CODE REVIEW

File: test_app/package.json [hunk 1]
```diff
@@ -18,6 +18,6 @@
     "echoMe": "bin/echoMe"
   },
   "dependencies": {
-    "binwrap": "^1.0.0"
+    "binwrap": ".."
```
Comment (avh4) [inline test_app/package.json]: This shouldn't need to change to a path, but probably should change to the next version number.  The way the test projects are set up here, `npm` is able to resolve it.

**Comment by michaelglass** [comment]
got it working with slightly different steps
```
$ git clean -dfx
$ cd test_app
$ npm install ../
$ cd ..
$ npm install
$ npm test
```

**Comment by michaelglass** [comment]
yup, that package doesn't work because of https://github.com/ZJONSSON/node-unzipper/issues/17

**Comment by rtfeldman** [comment]
so...maybe `unzip-stream` is the answer?

**Comment by michaelglass** [comment]
@rtfeldman yup, handled in https://github.com/avh4/binwrap/pull/12/commits/450c42cb3574c1d523d73ec5fe5b57550c5b4fee

**Comment by avh4** [comment]
I don't see any documentation saying that `unzip-stream` has the same API as `unzip`.  Has someone tested this to see if it actually can install a binary compressed in a zip?

**Comment by michaelglass** [comment]
closing in favor of #15

**Pull request closed without merging**